spec/models/work_order_spec.rb

```diff
@@ -1,237 +1,248 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
 RSpec.describe WorkOrder, type: :model do
-
   def make_uuid
     SecureRandom.uuid
   end
 
   before do
     @barcode_index = 100
-    stub_const("BillingFacadeClient", double('BillingFacadeClient'))
+    stub_const('BillingFacadeClient', double('BillingFacadeClient'))
+    stub_const('BrokerHandle', class_double('BrokerHandle'))
   end
 
   def make_barcode
     @barcode_index += 1
     "AKER-#{@barcode_index}"
   end
 
-  def make_set(size=6)
+  def make_set(size = 6)
     uuid = make_uuid
     s = double(:set, uuid: uuid, id: uuid, meta: { 'size' => size })
```

Comment:
If we're using an implicit return, could we rename 's' to something more descriptive, like 'set'?